config/schema/myparcel/order/postnl/nl_package.json

```diff
@@ -26,6 +26,11 @@
                     }
                 }
             }
+        },
+        "canBeMultiCollo": {
```

Comment:
Mocht je deze pr aanpassen, dan toch liever canHaveMultiCollo of iets anders dat in de lijn van de rest van de terminologie is. De 'Be' is hier toch te uniek achteraf gezien voor mijn gevoel.